dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh

```diff
@@ -46,8 +46,9 @@ if [ "${CHE_SERVER_URL}" == "" ]; then
     CHE_SERVER_URL="https://${CHE_SERVER_ROUTE_HOST}"
   fi
 fi
-
+DEFAULT_CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD=true
 oc set env buildconfig/keycloak-for-che CHE_SERVER_URL=${CHE_SERVER_URL}
+oc set env buildconfig/keycloak-for-che CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD=${CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD:-${DEFAULT_CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD}}
```

Comment:
you should be able to set the 2 env variables on the same line.

```
Usage:
  oc set env RESOURCE/NAME KEY_1=VAL_1 ... KEY_N=VAL_N [options]
```

Reply:
separator is just a space?

Reply:
yes, it seems.

Reply:
updated